### PR TITLE
refactor(StatusChatToolBar): open/close handlers get called if they a…

### PR DESCRIPTION
**Pull request by saledjenic**
…re set for `popupMenu` of `StatusChatToolBar` component

### CODE REVIEW

**Review by borismelnik — APPROVED** [review]

**Comment by PascalPrecht** [comment]
I see this was being merged into `refactoring/base_bc`
It looks like however, this should be done in `master` (as well). It also seems like this is either a fix or a feature, depending on how you look at it.

@saledjenic Would you mind sending this against `master` as well? `refactoring/base_bc` then needs to be rebased on top of `master`

**Comment by saledjenic** [comment]
@PascalPrecht sorry, if I had to wait for 2 approvals here as well, the point why I merged it after one approval is that I wanted to merge corresponding part in the `status-desktop` cause I need those changes for my further work. 

Since at the end (once we're done with refactor) this will go to the `master` branch definitely, does it necessary to do it now if there is no explicit need at the moment to have this in `master`?

**Comment by PascalPrecht** [comment]
> does it necessary to do it now if there is no explicit need at the moment to have this in master?

Yea that's a good point. I thought, since it's a small change and improves the current state, it'd be nice to see that improvement in `master` as well. Probably not a big deal if it isn't right now.

**Pull request merged**
